lua/terrortown/lang/en.lua

```diff
@@ -108,7 +108,7 @@ L.disg_menutitle = "Disguise control"
 L.disg_not_owned = "You are not carrying a Disguiser!"
 L.disg_enable = "Enable disguise"
 
-L.disg_help1 = "When your disguise is active, your name, health and karma do not show when someone looks at you. In addition, you will be hidden from a Detective's radar."
+L.disg_help1 = "When your disguise is active, your name, health and Karma do not show when someone looks at you. In addition, you will be hidden from a Detective's radar."
```

Comment:
What is the reason for capitalizing this?

Reply:
Because on the official TTT website Karma is always upper case. Therefore I replaced it in the whole translation